src/gitlab/configuration.ts

```diff
@@ -80,9 +80,9 @@ export class CiConfiguration extends Component {
    */
   public readonly defaultBeforeScript: string[] = [];
   /**
-   * A default list of files and directories to cache between jobs. You can only use paths that are in the local working copy.
+   * A default list of cache definitions (máx. 4) with the files and directories to cache between jobs. You can only use paths that are in the local working copy.
    */
-  public readonly defaultCache?: Cache;
+  public defaultCache?: Cache[];
```

Comment:
What we normally do here is turn the property into a private, mutable one and then expose it via a getter.
```suggestion
  private _defaultCache?: Cache[];
  
  public get defaultCache: Cache[] | undefined {
    return this._defaultCache;
  }
```